src/ensembl/src/content/app/entity-viewer/gene-view/components/gene-view-sidebar/external-references/GeneExternalReferences.tsx

```diff
@@ -113,15 +113,16 @@ const buildCrossReferenceGroups = (crossReferences: CrossReference[]) => {
 const GeneExternalReferences = () => {
   const params: EntityViewerParams = useParams();
 
-  const entityId = params.entityId?.split(':').pop();
+  const { entityId, genomeId } = params;
+
+  const stableId = entityId?.split(':').pop();
 
-  // TODO: The genomeId is temporarily hardcoded here as Thoas does ot have date for homo_sapiens_GCA_000001405_27.
```

Comment:
erm, web instance of thoas doesn't have `homo_sapiens_GCA_000001405_27 ` either.

Reply:
Yes, you are right @azangru  web-thoas doesn't have _homo_sapiens_GCA_000001405_27_  but it has **_homo_sapiens_GCA_000001405_28_** and application team thoas also has **_homo_sapiens_GCA_000001405_28_**

Reply:
Another ugly hack to use `homo_sapiens_GCA_000001405_28 ` in place of `homo_sapiens_GCA_000001405_27`  ?

Reply:
> Another ugly hack to use homo_sapiens_GCA_000001405_28 in place of homo_sapiens_GCA_000001405_27 ?

🤷‍♂️ 

@kamaldodiya is there anything stopping us from loading homo_sapiens_GCA_000001405_27 in web's instance of thoas?

Reply:
@azangru  AFAIK the production database from where we are getting the raw data (JSON dumps ) has GRCh38.p13 ( **homo_sapiens_GCA_000001405_28** )

Reply:
So annoying!

Can you pretend that it's homo_sapiens_GCA_000001405_27 when you are saving it in the DB?

Reply:
It seems that we'll have to have the ugly hack somewhere; and of course we as frontend developers would rather the hack were not in the frontend code.

Reply:
Why can't we switch to use homo_sapiens_GCA_000001405_28 rather than putting hack ? Putting hack is easier but it will put web and application teams thoas out of sync.  I think we should get rid of the problem rather than introducing it in thoas. Is it major refactoring in Ensembl-client ? I know it will need some changes in genome-search-api, not sure about 2020-server

Reply:
> Why can't we switch to use homo_sapiens_GCA_000001405_28

I have a suspicion that we may be blocked by genome browser. It probably does not have data files for _28; and probably hasn't updated the code to switch to the newly generated data files.

It may be possible, of course, to make genome browser pretend that its current data files for human are _28. Perhaps the only change required for this would be renaming of the data files. But it's a question to @ens-ds23.